docs/testcases/UI/System_Management/2232_NAT_Gateway/2232_6.md

```diff
@@ -0,0 +1,24 @@
+# Add route with resolved IP to the route without resolved IP - negative case
+
+Test verifies that it isn't possible to add route with resolved IP to the existing route without resolved IP.
+
+**Prerequisites**:
+- Admin user
+- Perform [2232_3](2232_3.md) case
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Click the **SYSTEM MANAGEMENT** tab | |
+| 4 | Click the **NAT GATEWAY** tab | |
+| 5 | Click **Add Route** button | |
+| 6 | Input `yahoo.com` into the ***Server name:*** field |  |
+| 7 | Mark ***Specify IP address*** checkbox |  |
+| 8 | Input `443` into the ***Port:*** field | |
```

Comment:
Suggest to add a verification here - whether the gateway with the same server name and port already exists. And if yes - try to use another port in this case, for example